active-directory-dotnet-native-uwp-v2/App.xaml.cs

```diff
@@ -106,8 +96,8 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
         //   - for any Work or School accounts, use organizations
         //   - for any Work or School accounts, or Microsoft personal account, use common
         //   - for Microsoft Personal account, use consumers
-        private static string ClientId = "0b8b0665-bc13-4fdc-bd72-e0227b9fc011";
-        private static string Tenant = "common";
+        private const string ClientId = "0b8b0665-bc13-4fdc-bd72-e0227b9fc011";
```

Comment:
Same thing. @andretms : does changing this affect the string replacements by the portal?